mypy/checkexpr.py

```diff
@@ -705,7 +705,7 @@ def check_call(self,
             return self.check_union_call(callee, args, arg_kinds, arg_names, context, arg_messages)
         elif isinstance(callee, Instance):
             call_function = analyze_member_access('__call__', callee, context,
-                                                  False, False, False, self.msg,
+                                                  False, False, True, self.msg,
```

Comment:
Not new to your diff, but it would be nice if we could use keyword args here so it's clear what all these bools mean.

Reply:
Did it, thanks!